pkgs/tools/package-management/cargo-license/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, rustPlatform, fetchFromGitHub }:
+rustPlatform.buildRustPackage rec {
+  name = "cargo-license-${version}";
+  version = "0.2.0";
+
+  src = fetchFromGitHub {
+    owner = "onur";
+    repo = "cargo-license";
+    rev = "v${version}";
+    sha256 = "0xxgl9d695ncrxz29125wag285dwxpwc3fym0ixgj5fqbnkbx75g";
+  };
+
+  cargoPatches = [ ./add-Cargo.lock.patch ];
+
+  cargoSha256 = "0jc84v8fxzzyfkcnfr9vrdblw5vdk54nzpch5lcarzfsarncqxw7";
+
+  meta = with lib; {
```

Comment:
`homepage` please :)
https://github.com/onur/cargo-license

Reply:
Oh yes, I'll add one. Thanks for pointing this out.

Reply:
Fixed in https://github.com/NixOS/nixpkgs/pull/63772